src/ui/components/Assist/Assist.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useEffect, useState } from "react";
+
+import Icon from "replay-next/components/Icon";
+import useLocalStorage from "replay-next/src/hooks/useLocalStorage";
+import { setViewMode } from "ui/actions/layout";
+import { setSelectedPrimaryPanel } from "ui/actions/layout";
+import Events from "ui/components/Events";
+import { shouldShowDevToolsNag } from "ui/components/Header/ViewToggle";
+import Confetti from "ui/components/shared//Confetti";
+import hooks from "ui/hooks";
+import { Nag } from "ui/hooks/users";
+import { useDismissNag } from "ui/hooks/users";
+import { UserInfo } from "ui/hooks/users";
+import { useTestInfo } from "ui/hooks/useTestInfo";
+import { getViewMode } from "ui/reducers/layout";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+import { ViewMode } from "ui/state/layout";
+import {
+  shouldShowAddComment,
+  shouldShowAddCommentToLine,
+  shouldShowAddCommentToNetworkRequest,
+  shouldShowAddCommentToPrintStatement,
+  shouldShowAddUnicornBadge,
+  shouldShowBreakpointAdd,
+  shouldShowBreakpointEdit,
+  shouldShowConsoleNavigate,
+  shouldShowExploreSources,
+  shouldShowInspectComponent,
+  shouldShowInspectElement,
+  shouldShowJumpToCode,
+  shouldShowJumpToEvent,
+  shouldShowLaunchCommandPalette,
+  shouldShowQuickOpenFile,
+  shouldShowRecordReplay,
+  shouldShowSearchSourceText,
+  shouldShowTour,
+  shouldShowUseFocusMode,
+} from "ui/utils/onboarding";
```

Comment:
These imports can be reduced too because the usage of them below can be removed:
```suggestion
import {
  shouldShowAddComment,
  shouldShowAddUnicornBadge,
  shouldShowBreakpointEdit,
  shouldShowConsoleNavigate,
  shouldShowExploreSources,
  shouldShowInspectElement,
  shouldShowJumpToCode,
  shouldShowJumpToEvent,
  shouldShowLaunchCommandPalette,
  shouldShowRecordReplay,
  shouldShowSearchSourceText,
  shouldShowTour,
  shouldShowUseFocusMode,
} from "ui/utils/onboarding";
```